guides/embedders/mistral.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Semantic Search with Mistral Embedding - Meilisearch documentation
+description: This guide will walk you through the process of setting up Meilisearch with Mistral embeddings to enable semantic search capabilities.
+---
+
+# Semantic Search with Mistral Embedding
+
+## Introduction
+
+This guide will walk you through the process of setting up Meilisearch with Mistral embeddings to enable semantic search capabilities. By leveraging Meilisearch's AI features and Mistral's embedding API, you can enhance your search experience and retrieve more relevant results.
+
+## Requirements
+
+To follow this guide, you'll need:
+
+- A Meilisearch Cloud project running version 1.6 or above with the Vector store activated.
```

Comment:
You're using the v1.10 API for REST API-based embedders, so you should recommend version >= 1.10

Reply:
I already updated this :)